packages/webhooks/tasks/sendWebhook.spec.js

```diff
@@ -48,37 +85,150 @@ const SendWebhookTests = (appName, actorsInitializer, userCreator, userDestroyer
+                                expect.objectContaining({ id: lastUser.id }),
+                            ]))
+                            expect(STABLE_CALLS.flat()).toHaveLength(5)
+                        },
+                    ],
+                    [
+                        'create',
+                        { create: true, update: false, delete: false },
+                        () => {
+                            // NOTE: DELETED USER MUST BE IN SENT DATA
+                            expect(STABLE_CALLS[0]).toEqual(expect.arrayContaining([
+                                expect.objectContaining({ id: firstUser.id }),
+                                expect.objectContaining({ id: deletedUser.id }),
+                                expect.objectContaining({ id: lastUser.id }),
+                            ]))
+                            expect(STABLE_CALLS.flat()).toHaveLength(3)
+                        },
+                    ],
+                    [
+                        'update',
+                        { create: false, update: true, delete: false },
+                        () => {
+                            // NOTE: DELETED USER MUST BE IN SENT DATA
+                            expect(STABLE_CALLS[0]).toEqual(expect.arrayContaining([
+                                expect.objectContaining({ id: firstUser.id }),
+                                expect.objectContaining({ id: deletedUser.id }),
+                                expect.objectContaining({ id: lastUser.id }),
+                            ]))
+                            expect(STABLE_CALLS[1]).toEqual(expect.arrayContaining([
+                                expect.objectContaining({ id: firstUser.id }),
+                            ]))
+                            expect(STABLE_CALLS[2]).toEqual(expect.arrayContaining([
+                                expect.objectContaining({ id: lastUser.id }),
+                            ]))
+                            expect(STABLE_CALLS.flat()).toHaveLength(5)
+                        },
+                    ],
+                    [
+                        'delete',
+                        { create: false, update: false, delete: true },
+                        () => {
+                            // NOTE: DELETED USER MUST BE IN SENT DATA
+                            expect(STABLE_CALLS[0]).toEqual(expect.arrayContaining([
+                                expect.objectContaining({ id: deletedUser.id }),
+                            ]))
+                            expect(STABLE_CALLS[1]).toEqual(expect.arrayContaining([
+                                expect.objectContaining({ id: lastUser.id }),
+                            ]))
+                            expect(STABLE_CALLS.flat()).toHaveLength(2)
+                        },
+                    ],
+                    [
+                        'create and delete',
+                        { create: true, update: false, delete: true },
```

Comment:
If you want to work with multiple operations at once, you should probably pass operation as a part of payload